src/main/java/uk/gov/companieshouse/acsp/service/OfficerService.java

```diff
@@ -0,0 +1,89 @@
+package uk.gov.companieshouse.acsp.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.util.CollectionUtils;
+import org.springframework.util.ObjectUtils;
+import org.springframework.web.util.UriTemplate;
+import uk.gov.companieshouse.acsp.Exception.ServiceException;
+import uk.gov.companieshouse.acsp.sdk.ApiClientService;
+import uk.gov.companieshouse.api.ApiClient;
+import uk.gov.companieshouse.api.error.ApiErrorResponseException;
+import uk.gov.companieshouse.api.handler.exception.URIValidationException;
+import uk.gov.companieshouse.api.handler.officers.request.OfficersList;
+import uk.gov.companieshouse.api.model.officers.OfficersApi;
+import uk.gov.companieshouse.logging.Logger;
+import uk.gov.companieshouse.logging.LoggerFactory;
+
+import java.io.IOException;
+
+import static uk.gov.companieshouse.acsp.AcspApplication.APP_NAMESPACE;
+import static uk.gov.companieshouse.acsp.util.Constants.TRUE;
+
+@Service
+public class OfficerService {
+
+    private final ApiClientService apiClientService;
```

Comment:
does this need to be autowired?

Reply:
Yes, It  is autowired using constructor injection. Line 33